operations/helm/charts/mimir-distributed/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 6.1.0-weekly.366
-appVersion: r366
+version: 6.0.0
```

Comment:
Just flagging that this version decrease looks strange to me, but i see the bot bumped it from `6.0.0-rc.0` to a 6.1 prerelease yesterday, so i guess that was erroneous :eyes: